app/assets/stylesheets/components/rad-notification/_settings.scss

```diff
@@ -0,0 +1,5 @@
+$rad-notification-radius: 5px;
+$color-rad-notification-warning: #F89E54;
```

Comment:
Definitely fine as is but I think there's a slight misunderstanding.

We should still have the 'local' definition of this variable in the component's `_settings.scss` file, but it should be referencing a global colour that lives in the project's `variables.scss`. Does that make sense?